### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by vlebourl**

### CODE REVIEW

File: custom_components/tahoma/translations/en.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-        },
-        "step": {
-            "user": {
-                "data": {
-                    "username": "Username",
-                    "password": "Password"
-                },
-                "description": "Enter the credentials you use for the Tahoma app or the tahomalink.com platform."   
-            }
+  "title": "Tahoma",
```
Comment (iMicknl) [inline custom_components/tahoma/translations/en.json:2]: Could you remove the `title` from all files? I understood this is not mandatory and will be filled by the integration name (Somfy TaHoma). Could you try this?

Comment (vlebourl) [reply custom_components/tahoma/translations/en.json]: title isn't necessary indeed. Deduced title is "Somfy Tahoma" which is good.
It seems we can't use common keys in French (and I suspect other non english languages?)
image

Comment (iMicknl) [reply custom_components/tahoma/translations/en.json]: I am not sure how the common keys part works to be honest... You can translate them for now.

Comment (vlebourl) [reply custom_components/tahoma/translations/en.json]: did it [here](https://github.com/iMicknl/ha-tahoma/pull/45/commits/942085efcd78a47f1e2815888261953fbaad8b56)

**Comment by iMicknl** [comment]
Merci!

**Comment by vlebourl** [comment]
graag gedaan

**Pull request merged**